src/model/Som.py

```diff
@@ -29,6 +32,18 @@ def __init__(self, id_som, titulo, caminho, volume):
         self.titulo = titulo
         self.volume = 100
 
+        self.dick = {
+            "título": self.titulo,
+            "caminho": self.caminho,
+            "formato": self.id,
+            "volume": self.volume,
+        }
```

Comment:
Acho que as alterações que fizemos acabaram não sendo salvas. Lembra que a gente tinha tirado essa transformação e colocado na função, pra que ela só seja processada quando a função for chamada?